modules/ROOT/pages/about-mule-message.adoc

```diff
@@ -8,16 +8,16 @@ A Mule message is the part of the Mule event that serves as a container for mess
 
 The Mule message is immutable, so every change to a Mule message results in the creation of a new instance. Each processor in a flow that receives a message returns a new Mule message consisting of these parts:
 
-* A message `payload`, which contains the body of the message. For example: the content of a file, a record from a database, or the response to a REST or Web Service request.
-* Message `attributes`, which are metadata associated with the payload.
+* A message `payload`, which contains the body of the message. For example: the content of a file, the HTTP body, a record from a database, or the response to a REST or Web Service request.
+* Message `attributes`, which are metadata associated with the payload, such as HTTP headers and query parameters.
 
 image::mule-message-structure-82af1.png[]
 
-A Mule message is created as part of a xref:about-mule-event.adoc[Mule event] during the process in which a Mule event source in a Mule flow triggers a flow to start, as when an HTTP Listener component receives a response or each time the Scheduler component triggers an execution of the flow.
+Execution of a Mule event source initiates a process that triggers the creation of a Mule message, places the message in a Mule event, and starts the Mule flow that contains the event source. For example, this process begins each time an  xref:connectors::http/http-listener-ref.adoc[HTTP Listener] source receives a request or the xref:scheduler-concept.adoc[Scheduler] executes. 
```

Comment:
I would remove "Execution of" and I think all the clarifications about the mule flow "that contains the event source" makes it more difficult to understand and has no use for the end user. The flow doesn't start in that moment but is executed. I would simply  add this:
"A mule event source triggers a process which involves the creation of an event and the execution of the flow associated to it. For example, this process begins.."

Reply:
works for me, thx!

Reply:
If you define that an event has a message, it implies from the event source, the created event will have a message. I think there is nothing strictly incorrect about the paragraph so it's ok for me if it seems clearer for the users.

Reply:
```suggestion
A Mule event source triggers a process that creates a Mule event and executes the Mule flow that contains the event source. A Mule message is part of this event. For example, this process begins each time an  xref:connectors::http/http-listener-ref.adoc[HTTP Listener] source receives a request or the xref:scheduler-concept.adoc[Scheduler] executes. 
```